src/app/project-overview/style.scss

```diff
@@ -27,3 +27,13 @@
     padding: 0 30px;
   }
 }
+
+.first-visit-overview-message {
+  margin-bottom: 20px;
+  padding: 10px;
+
+  i .km-icon-close {
```

Comment:
Update alignment according to design + `x` icon

<img width="1192" alt="icon" src="https://user-images.githubusercontent.com/17727069/179201679-b9f35092-b7a8-4390-824a-9c4e6956dcb8.png">